spec/integration/domchange_spec.rb

```diff
@@ -3,8 +3,7 @@
   scenario 'the DOM is ready' do
     visit '/domchange'
 
-    expect(page.driver.console_messages.map { |m| m[:message] })
-      .to eq(['uic:domchange was fired'])
+    expect(JS_LOGGER.string).to eq("uic:domchange was fired\n")
```

Comment:
Curious, if this waits for the page to completely load. If not, it may randomly fail. Maybe, `expect(page).to have_content('anything')`?